fiasco/__init__.py

```diff
@@ -1,11 +1,11 @@
 """
-A Python interface to the CHIANTI atomic database
+fiasco: A Python interface to the CHIANTI atomic database
 """
-from fiasco.collections import *
-from fiasco.elements import *
-from fiasco.fiasco import *
-from fiasco.ions import *
-from fiasco.levels import *
+from fiasco.collections import IonCollection
+from fiasco.elements import Element
+from fiasco.fiasco import list_elements, list_ions, proton_electron_ratio
+from fiasco.ions import Ion
+from fiasco.levels import Level, Transitions
```

Comment:
Why is this not allowed? This defeats the purpose of being able to use `__all__` to define what the top level API should be for each of these modules.

Reply:
This is allowed, I just wanted to make it super explicit. I can undo this.

Reply:
Ok I think I understand the logic here. I do like the idea of more explicitly defining the top level namespace here.